src/rsa.rs

```diff
@@ -227,6 +227,7 @@ fn parse_public_key<'a>(input: untrusted::Input<'a>) ->
 #[cfg(feature = "rsa_signing")]
 pub struct RSAKeyPair {
     rsa: std::boxed::Box<RSA>,
+    blinding: std::sync::Mutex<*mut BN_BLINDING>,
```

Comment:
Yeah, my assumption is that we'll always be passing `blinding` pointer by value when crossing the FFI boundary, so there's no need to heap allocate, unlike the `rsa` field.